packages/terra-form-select/src/native-select/NativeSelect.module.scss

```diff
@@ -34,7 +34,10 @@
       background-color: var(--terra-form-select-native-focus-background-color, #fff);
```

Comment:
Do we still need the &[data-focus-interaction='keyboard'] styles above?  The only difference now appears to be the box-shadow.  Should that be part of this selector as well, or is it no longer applicable.  Also, based on the selector being used, could this actually be replaced with the ":focus" pseudo class selector?

Reply:
+1

Reply:
1) Removed duplicate [data-focus-interaction='keyboard'] selector as suggested. 
2) Box-shadow property is required in default theme since we are applying box-shadow in Fusion theme. 
3) We can't replace [data-focus-interaction='keyboard'], [data-focus-interaction='mouse'] selectors with ":focus" pseudo class selector because these styles has been added for div element and not for select element, and we are getting the above keyboard and mouse attributes on the same div element. Based on the attributes of div element we are applying the focus styles.
<img width="1728" alt="Screenshot 2023-08-10 at 5 58 29 PM" src="https://github.com/cerner/terra-core/assets/127083430/7b4e99cf-6777-41cc-b887-1d8b04f55bb6">

Reply:
we can still remove mouse and keyboard values from data attrs. updated with commit [9f570493](https://github.com/cerner/terra-core/pull/3860/commits/65689d4dd67ae4293cff83ad933a83289f570493)